app/scripts/modules/core/src/config/settings.ts

```diff
@@ -47,6 +47,7 @@ export interface IFeatures {
   pipelineTemplates?: boolean;
   versionedProviders?: boolean;
   travis?: boolean;
+  managedServiceAccountsEnabled?: boolean;
```

Comment:
This doesn't match the `settings.js` value you've configured - I think it should be `managedServiceAccounts`.